doc/docker/jenkinsnode/run-make

```diff
@@ -0,0 +1,4 @@
+#!/bin/sh
+
+set -ex
+run-nice make -j 1 $* VERBOSE=1
```

Comment:
Set higher?

Reply:
That really depends on the machine you put the node on, the Readme tells you that you may adjust this to a higher setting if desired.

Reply:
Okay, I thought that the tutorial is written for as you did it on the v2. I have written a separate issue for this.